tools/fetch_fasta_from_ncbi/retrieve_fasta_from_NCBI.py

```diff
@@ -168,17 +186,21 @@ def efetch(self, db, query_key, webenv):
             try:
                 response = urllib2.urlopen(req)
                 fasta = response.read()
-                if ("Resource temporarily unavailable" in fasta) or (not fasta.startswith(">") ):
+                if (response.getcode() != 200) or ("Resource temporarily unavailable" in fasta) or ("Error" in fasta) or (not fasta.startswith(">") ):
```

Comment:
Can you indent the line to make it shorter ? see https://www.python.org/dev/peps/pep-0008/#indentation